pom.xml

```diff
@@ -1,338 +1,96 @@
-		</repository>
-	</distributionManagement>
-</project>
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.openmrs.maven.plugins</groupId>
+    <artifactId>openmrs-sdk-maven-plugin-main</artifactId>
+    <packaging>pom</packaging>
+    <version>2.1.3-SNAPSHOT</version>
+
+    <name>OpenMRS SDK Maven Plugin</name>
+    <description>OpenMRS SDK allows for rapid development of OpenMRS modules.</description>
+    <url>https://wiki.openmrs.org/display/docs/OpenMRS+SDK</url>
+
+    <modules>
+        <module>openmrs-sdk-maven-plugin</module>
+        <module>openmrs-sdk-maven-plugin-integration-test</module>
+    </modules>
+
+    <licenses>
+        <license>
+            <name>Mozilla Public License 2.0 with Healthcare Disclaimer</name>
+            <url>http://openmrs.org/license</url>
+        </license>
+    </licenses>
+
+    <developers>
+        <developer>
+            <name>Rafal Korytkowski</name>
+            <email>rafal@openmrs.org</email>
+            <organization>OpenMRS</organization>
+            <organizationUrl>http://openmrs.org/</organizationUrl>
+        </developer>
+        <developer>
+            <name>Dmytro Trifonov</name>
+            <email>dmytro.trifonov@openmrs.org</email>
+            <organization>OpenMRS</organization>
+            <organizationUrl>http://openmrs.org/</organizationUrl>
+        </developer>
+    </developers>
+
+    <scm>
+        <connection>scm:git:https://github.com/openmrs/openmrs-contrib-sdk-maven-plugin</connection>
+        <developerConnection>scm:git:https://github.com/openmrs/openmrs-contrib-sdk-maven-plugin</developerConnection>
+        <url>https://github.com/openmrs/openmrs-contrib-sdk-maven-plugin</url>
+        <tag>HEAD</tag>
+    </scm>
+
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <tomcat.version>7.0.67</tomcat.version>
+    </properties>
+
+    <dependencies>
+        <!-- testing -->
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.hamcrest</groupId>
+            <artifactId>hamcrest-core</artifactId>
+            <version>1.3</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.hamcrest</groupId>
+            <artifactId>hamcrest-library</artifactId>
+            <version>1.3</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
```

Comment:
It's not recommended to specify dependencies in main pom. If you want to share the version then put them in dependencyManagement section in the main pom.